libs/backend/src/cast_vote_records/export.ts

```diff
@@ -423,6 +425,76 @@ async function exportSignatureFileToUsbDrive(
   return ok();
 }
 
+/**
+ * Updates the creation timestamp of a directory and its children files (assuming no
+ * sub-directories) using the one method guaranteed to work:
+ * ```
+ * cp -r <directory-path> <directory-path>-temp
+ * rm <directory-path>
+ * mv <directory-path>-temp <directory-path>
+ * ```
+ */
+async function updateCreationTimestampOfDirectoryAndChildrenFiles(
+  directoryPath: string
+): Promise<void> {
+  const tempDirectoryPath = `${directoryPath}-temp`;
+  await fs.mkdir(tempDirectoryPath);
+  const fileNames = (await fs.readdir(directoryPath, { withFileTypes: true }))
+    .filter((entry) => entry.isFile())
+    .map((entry) => entry.name);
+  for (const fileName of fileNames) {
+    await fs.copyFile(
+      path.join(directoryPath, fileName),
+      path.join(tempDirectoryPath, fileName)
+    );
+  }
+  await fs.rm(directoryPath, { recursive: true });
+  await fs.rename(tempDirectoryPath, directoryPath);
```

Comment:
In the comment's description of this function, you explain what I would expect - copy and rename directory. But in the function itself you're iterating and copying each file individually rather than copying the directory recursively. Is there a reason for this?

Reply:
Ah yes good q, meant to leave a note about this. `fs.cp`, which supports a `recursive` param, is [experimental](https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options), so I didn't want to use it

Reply:
Added a clarifying comment in https://github.com/votingworks/vxsuite/pull/3972/commits/bc83adca281107c4fd81c5408af0802da159ba38